src/components/ui/OakPrimaryInvertedButton/OakPrimaryInvertedButton.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ElementType } from "react";
+
+import {
+  InternalShadowRectButton,
+  InternalShadowRectButtonProps,
+} from "@/components/ui/InternalShadowRectButton";
+import { PolymorphicPropsWithoutRef } from "@/components/utils/polymorphic";
+
+export type OakPrimaryInvertedButtonProps = Omit<
+  InternalShadowRectButtonProps,
+  | "defaultBorderColor"
+  | "defaultBackground"
+  | "defaultTextColor"
+  | "hoverBackground"
+  | "hoverBorderColor"
+  | "hoverTextColor"
+  | "disabledBackground"
+  | "disabledBorderColor"
+  | "disabledTextColor"
+>;
+
+export const OakPrimaryInvertedButton = <C extends ElementType = "button">({
+  element,
+  ...rest
+}: OakPrimaryInvertedButtonProps & PolymorphicPropsWithoutRef<C>) => {
+  return (
+    <InternalShadowRectButton
+      element={element ?? "button"}
+      defaultBorderColor="bg-btn-secondary"
+      defaultBackground="bg-btn-secondary"
+      defaultTextColor="text-primary"
+      hoverBackground="bg-btn-secondary"
```

Comment:
I don't understand how this isn't causing the background to change colour on hover ?

Reply:
it is set to the same background colour in default state and on hover so it is not suppose to change

Reply:
`bg-btn-secondary` is white isn't it?

<img width="1069" alt="Screenshot 2024-01-31 at 11 38 46" src="https://github.com/oaknational/oak-components/assets/122096/5c67d266-4738-4406-a31f-a755995931b1">

might be worth setting a default background on the story to make that clearer?

```typescript
  parameters: {
    backgrounds: {
      default: "light",
    },
  }
```